package.json

```diff
@@ -81,6 +81,9 @@
   "optionalDependencies": {
     "@rollup/rollup-linux-x64-gnu": "^4.9.6"
   },
+  "overrides": {
+    "nwsapi": "2.2.2"
+  },
```

Comment:
Had to utilize an override here, as there's a problem with the latest version of `nwsapi` and the `:has` selector ([some context](https://github.com/dperini/nwsapi/issues/123)), which causes some tests to fail (such as `TreeView.test.tsx), since there's usage of `:has` in some selectors. 

When this is resolved we can remove it, but for now it's needed in order for us to bump `@figma/code-connect`.